app/db/migrations/versions/~2024_10_16_1347-da30746cec39_.py

```diff
@@ -0,0 +1,40 @@
+"""Add MultiInputField to ComponentType Enum and columns children and schema to Component. Add column options to Page
+
+Revision ID: da30746cec39
+Revises: ca61d3b746f6
+Create Date: 2024-10-16 13:47:16.968837
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'da30746cec39'
+down_revision = 'ca61d3b746f6'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+        
+    op.execute("ALTER TYPE componenttype ADD VALUE 'MULTI_INPUT_FIELD';")
```

Comment:
In multiinputfield we have to have RepeatingFieldPageController as the controller & 
`"options": {
                "summaryDisplayMode": {
                    "samePage": true,
                    "separatePage": false,
                    "hideRowTitles": false
                },
                "customText": {
                    "samePageTitle": "Your secured match funding"
                }
            },`

options to show up the table is it already handling ?

Reply:
Yes it's added in this same pr :)